evidently/analyzers/test_data_drift_analyzer.py

```diff
@@ -0,0 +1,70 @@
+import json
+import pytest
+from pandas import DataFrame
+
+from evidently import ColumnMapping
+from evidently.analyzers.data_drift_analyzer import DataDriftAnalyzer
+from evidently.options import DataDriftOptions, OptionsProvider
+from evidently.utils import NumpyEncoder
+
+
+@pytest.fixture
+def analyzer() -> DataDriftAnalyzer:
+    options_provider: OptionsProvider = OptionsProvider()
+    options_provider.add(DataDriftOptions())
+    analyzer = DataDriftAnalyzer()
+    analyzer.options_provider = options_provider
+    return analyzer
+
+
+def _check_feature_metrics(feature_metric: dict, feature_type: str):
+    assert 'current_small_hist' in feature_metric
+    assert 'feature_type' in feature_metric
+    assert feature_metric['feature_type'] == feature_type
+    assert 'p_value' in feature_metric
+    assert 'ref_small_hist' in feature_metric
+
+
+def test_data_drift_analyzer_as_dict_format(analyzer: DataDriftAnalyzer) -> None:
```

Comment:
As mentioned in the `as_dict` comment: these tests should be moved to `profile_section`.

Reply:
done